net/net.go

```diff
@@ -271,3 +272,10 @@ func getIOCountersAll(n []IOCountersStat) ([]IOCountersStat, error) {
 
 	return []IOCountersStat{r}, nil
 }
+
+// IsLittleEndian checks if the current platform uses little-endian.
+// copied from https://github.com/ntrrg/ntgo/blob/v0.8.0/runtime/infrastructure.go#L16 (MIT License)
+func IsLittleEndian() bool {
```

Comment:
do we want this function to be exported here?

Reply:
Absolutely no! I have moved to `internal/common` because it may be used from other package. Thank you for pointing out!!